.github/workflows/build.yml

```diff
@@ -2,9 +2,6 @@ name: Node.js CI
 
 on:
   push:
```

Comment:
I _think_ building on each push is the right thing to do.  Otherwise, build and test is only done on `master` commit.